docs/services/navigation/_index.md

```diff
@@ -620,8 +620,8 @@ The following {{< glossary_tooltip term_id="model" text="models" >}} of the [mov
 An example of an `AngularVelocity` reading:
 
 ``` go
-ang_vel, err := imu.AngularVelocity{context.Background, nil}
-imu.AngularVelocity{"_type":"angular_velocity","x":0,"y":0,"z":1}
+// angular_velocity is an AngularVelcoity{X:1, Y:2, Z:3}
+angular_velocity, err := imu.AngularVelocity{context.Background, nil}
```

Comment:
```suggestion
angular_velocity, err := imu.AngularVelocity(context.Background, nil)
```

Reply:
Done, thank you! My bad here-- I normally know it's supposed to be () if method call but just wasn't thinking properly. Thank you for catching